vendordeps/REVLib.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
I am dumb and pressed approved instead of needs changes. They require there will be a comment to request changes so here it is.